pkg/controller/persistentvolume/persistentvolume_controller.go

```diff
@@ -73,6 +73,30 @@ const annBoundByController = "pv.kubernetes.io/bound-by-controller"
 // provisioning to choose a particular storage class (aka profile).
 const annClass = "volume.alpha.kubernetes.io/storage-class"
 
+// This annotation is added to a PV that has been dynamically provisioned by
+// Kubernetes. It's value is name of volume plugin that created the volume.
```

Comment:
is this always the name of a volume plugin or is it the name of a provisioner or of a storage class?